src/main/java/org/fundacionjala/sfdc/pageobjects/Accounts/SFAMainPage.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.sfdc.pageobjects.Accounts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * SFAMainPage.java
+ * Class that represents the main page of accounts section.
+ */
+public class SFAMainPage extends SalesForceConnection{
+    private WebDriverWait wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)